scripts/access_control/Deploy_Granular_CCC_Guardian.s.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import '../../src/contracts/access_control/GranularGuardianAccessControl.sol';
+//import {GovernanceV3Ethereum} from 'aave-address-book/GovernanceV3Ethereum.sol';
+import {GovernanceV3Polygon} from 'aave-address-book/GovernanceV3Polygon.sol';
+import {GovernanceV3Avalanche} from 'aave-address-book/GovernanceV3Avalanche.sol';
+import {GovernanceV3BNB} from 'aave-address-book/GovernanceV3BNB.sol';
+import {GovernanceV3Gnosis} from 'aave-address-book/GovernanceV3Gnosis.sol';
+//import {MiscEthereum} from 'aave-address-book/MiscEthereum.sol';
+import {MiscPolygon} from 'aave-address-book/MiscPolygon.sol';
+import {MiscAvalanche} from 'aave-address-book/MiscAvalanche.sol';
+import {MiscBNB} from 'aave-address-book/MiscBNB.sol';
+import {MiscGnosis} from 'aave-address-book/MiscGnosis.sol';
+import '../BaseScript.sol';
+
+abstract contract BaseDeployGranularGuardian is BaseScript {
+  function AAVE_GUARDIAN() public view virtual returns (address);
```

Comment:
I would just do setters of all this params in constructor, and make it immutable, but it's not so important